custom/enikshay/reports/views.py

```diff
@@ -73,6 +75,23 @@ def duplicate_ids_report(request, domain, case_type):
     counts = Counter(case['readable_id'] for case in all_cases)
     bad_cases = [case for case in all_cases if counts[case['readable_id']] > 1]
 
+
+    for case in bad_cases:
+        form = CaseAccessorSQL.get_transactions(case['case_id'])[0].form
```

Comment:
This doesn't need to be correct 100% of the time - it's accurate for the typical use case (I checked against the production data), and that's really all I'm concerned with for the purposes of this.